src/push-workspace.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import chalk from 'chalk';
+import * as inquirer from 'inquirer';
+import { WorkspaceRepo, startNewBranch, commitChanges, pushChangesToGithub} from 'polymer-workspaces';
+
+export default async function run(reposToConvert: WorkspaceRepo[]) {
+
+  console.log(
+      chalk.dim('[1/5] ') + chalk.magenta(`Setting up push to GitHub...`));
+  const {commitMessage, branchName, forcePush} = (await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'branchName',
+      message: 'push to branch:',
+      default: 'polymer-modulizer-auto-generated',
+    },
+    {
+      type: 'confirm',
+      name: 'forcePush',
+      message:
+          'force push? (WARNING: This will overwrite any existing branch on github)',
```

Comment:
can you include the branch name from the previous prompt in the message? So it prints like:

'force push? (WARNING: This will overwrite any existing polymer-modulizer-auto-generated branch on GitHub)'

Also, capitalize GitHub